content/java/concepts/strings/terms/equalsIgnoreCase/equalsIgnoreCase.md

```diff
@@ -0,0 +1,46 @@
+---
+Title: '.equalsIgnoreCase()'
+Description: 'Compares strings in a case-insensitive manner.'
+Subjects:
+  - 'Computer Science'
+Tags:
+  - 'Methods'
+  - 'String'
+CatalogContent:
+  - 'learn-java'
+---
+
+The **`.equalsIgnoreCase()`** method compares strings and then then gives an output of true or false. It is not a case sensitive method.
+
+## Syntax
+
+```pseudo

```

Comment:
You haven't addressed my earlier content on the syntax block:

This more of an example. Syntax should be more general. Like this:

`boolean result = stringOne.equalsIgnoreCase(stringTwo);`

Then have a paragraph after the syntax example explaining what `stringOne` and `stringTwo` are, and how result is determined.